connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -186,7 +196,13 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
   def classloader: ClassLoader = artifactManager.classloader
 
   private[connect] def updateAccessTime(): Unit = {
-    lastRpcAccessTime = Some(System.currentTimeMillis())
+    lastAccessTime = System.currentTimeMillis()
+    logInfo(s"Session $key accessed, time $lastAccessTime.")
```

Comment:
Temporary log?

Reply:
I think it can stay as a logInfo, it happens once per RPC, and we have other logInfo logging at this volume (e.g. when an execution reattaches). This may be useful in default logs for debugging.

Reply:
That is a lot of logging. The other logging would be indication of this anyway (though it would be better to reduce that too).
But I am ok if you prefer it.

Reply:
When putting logging into Spark Connect execution I was guiding myself with what Thriftserver logs, and there we were putting log info logs as long as it was at most linear to the number of RPC calls. They proved quite useful in debugging many times.